WordCount/WordCountAPI/Controllers/WordCountController.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System.Collections.Generic;
+using Microsoft.AspNetCore.Mvc;
+using WordCount.JsonModels;
+
+namespace KnoxDatabaseLayer3.Controllers
+{
+    [ApiController]
```

Comment:
Controller looks clear and concise. Nice with comments on what will be implemented/put together at a later stage.